src/ansys/sound/core/sound_composer/source_spectrum.py

```diff
@@ -0,0 +1,240 @@
+# Copyright (C) 2023 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""Sound composer's spectrum source."""
+import os
+import warnings
+
+from ansys.dpf.core import Field, Operator
+from matplotlib import pyplot as plt
+import numpy as np
+from numpy import typing as npt
+
+from .._pyansys_sound import PyAnsysSoundException, PyAnsysSoundWarning
+from ._source_parent import SourceParent
+from .source_control_spectrum import SourceControlSpectrum
+
+ID_COMPUTE_LOAD_SOURCE_SPECTRUM = "sound_composer_load_source_spectrum"
+ID_COMPUTE_GENERATE_SOUND_SPECTRUM = "sound_composer_generate_sound_spectrum"
+
+
+class SourceSpectrum(SourceParent):
+    """Sound composer's spectrum source class.
+
+    This class create a spectrum source for the sound composer.
```

Comment:
TYPO: This class creates a spectrum source for the sound composer.

Reply:
@mattbennett72 I just pushed some updates, please refresh on your side (I believe I caught this typo).
I also changed sound composer to Sound Composer everywhere
FYI @ansaminard